core/src/main/res/values-night/color.xml

```diff
@@ -5,10 +5,10 @@
   <!--The same color values are defined for dark theme, but with values more appropriate for -->
   <!--low-luminance UIs. These colors will be used by night/themes.xml.-->
 
-  <color name="color_primary">@color/blue800</color>
+  <color name="color_primary">#42a5f5</color> <!--blue400-->
```

Comment:
These colors have a name so put their definition in the main colors file and reference them here. Adding context via well named variables is always better than comments.

eg in colors.xml
```
<color name="blue400">#42a5f5</color>
```
in night/colors
```
<color name="color_primary">@color/blue400</color>
```

Reply:
Done.
 Please review!